Misc/NEWS.d/next/Library/2023-03-22-16-15-18.bpo-102780.NEcljy.rst

```diff
@@ -0,0 +1,3 @@
+The `asyncio.timeout()` now works reliably even when performing cleanup due
+to task cancellation.  Previously the context manager could raise a
+`asyncio.CancelledError` instead of an `asyncio.TimeoutError` in such cases.
```

Comment:
```suggestion
:func:`asyncio.timeout` now works reliably even when performing cleanup due
to task cancellation.  Previously the context manager could raise a
:exc:`asyncio.CancelledError` instead of an :exc:`asyncio.TimeoutError` in such cases.
```